pkg/cloudprovider/aws/packing/nodecapacity.go

```diff
@@ -74,6 +77,41 @@ var (
 	}
 )
 
+func instanceTypeInfoToNodeCapacity(instanceTypeInfo ec2.InstanceTypeInfo) (*nodeCapacity, error) {
```

Comment:
Check out resource.MustParse. You shouldn't need to error afterwards.

Reply:
yeah I opted to use the error version to avoid the panic, but if we truly can't recover, maybe it's okay in this case? I usually try not to panic outside of pkg main.

Reply:
Yeah I follow the same, but with a bit more flexibility. If you can guarantee (in some way) that the data will be well formed (I think we can trust EC2 here), panicing can help keep the code clean. This is particularly important when go libraries put errors into functions when errors will only occur if you call them incorrectly, but won't error for other reasons (like a network call).

Reply:
I think `MustParse` would be ok here, if EC2 were returning bad results we can't really proceed anyway.